packages/main/test/specs/MessageStrip.spec.js

```diff
@@ -102,3 +102,18 @@ describe("ARIA Support", () => {
 	});
 });
 
+describe("Change properties dynamically", () => {
```

Comment:
Just to let you know (as it's not written anywhere), it's good to avoid introducing too many describe statements especially for one new test, as it also comes with one more page loading which is expensive and gradually adds up to make test execution longer. 

You can add the test in the describe (API) section, it's perfectly fine to be there.